src/proxy/pool.rs

```diff
@@ -64,7 +63,6 @@ struct PoolState {
     connected_pool: Arc<pingora_pool::ConnectionPool<ConnClient>>,
     // this must be an atomic/concurrent-safe list-of-locks, so we can lock per-key, not globally, and avoid holding up all conn attempts
     established_conn_writelock: flurry::HashMap<u64, Option<Arc<Mutex<()>>>>,
-    close_pollers: futures::stream::FuturesUnordered<task::JoinHandle<()>>,
```

Comment:
Ah yeah this does nothing ATM.

Was originally `await`-ing all the join handles in `impl Drop for PoolState` after signaling to make sure all conns evict, but I don't think that's strictly necessary.